src/locales/ko/mystery-encounters/clowning-around-dialogue.json

```
@@ -3,28 +3,28 @@
   "speaker": "어릿광대",
   "intro_dialogue": "거기 있는 어리버리한 바보씨, 화려한 배틀을 맞이할 각오를 하시길!\n이 싸움꾼 광대가 당신을 쓰러트리겠습니다!",
   "title": "어릿광대의 장난",
-  "description": "이번 상대는 뭔가 이상합니다.\n어릿광대가 당신을 전투에 끌어들이고 싶어하는 것 같은데, 무슨 목적일까요?\n\n{{blacephalonName}}[[은]] 더욱 수상합니다.\n마치 @[TOOLTIP_TITLE]{이상한 타입과 능력}을 가진 것 같습니다.",
+  "description": "이번 상대는 뭔가 이상합니다.\n어릿광대가 당신을 배틀에 끌어들이고 싶어하는 것 같은데, 무슨 목적일까요?\n\n{{blacephalonName}}[[은]] 더욱 수상합니다.\n마치 @[TOOLTIP_TITLE]{이상한 타입과 능력}을 가진 것 같습니다.",
   "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
       "label": "어릿광대와 싸운다",
-      "tooltip": "(-) 이상한 배틀\n(?) 포켓몬의 특성 조정",
+      "tooltip": "(-) 이상한 배틀\n(?) 포켓몬 1마리의 특성 조정",
       "selected": "당신의 불쌍한 포켓몬들이 한심한 공연을 할 준비가 된 것 같군요!",
-      "apply_ability_dialogue": "센세이셔널한 쇼였습니다!\n당신의 멋진 기지에는 이 센세이셔널한 보상이 잘 어울릴 것 같군요!",
+      "apply_ability_dialogue": "센세이셔널한 쇼였습니다!\n당신의 멋진 기지에는 특별한 보상이 어울릴 것 같군요!",
       "apply_ability_message": "어릿광대가 포켓몬 한 마리의 특성을 영구적으로 {{ability}}로 교환해 주겠다고 제안했다!",
       "ability_prompt": "포켓몬의 특성을 영구적으로 {{ability}} 특성으로 바꾸시겠습니까?",
       "ability_gained": "@s{level_up_fanfare}{{chosenPokemon}}[[는]] {{ability}} 특성을 얻었다!"
     },
     "2": {
       "label": "도발에 응하지 않는다",
-      "tooltip": "(-) 광대가 화를 냄\n(?) 포켓몬의 아이템 조정",
+      "tooltip": "(-) 포켓몬 1마리의 아이템 조정",
       "selected": "한심한 겁쟁이로군요? 즐거운 배틀을 거부하다니!\n이러면 한 방 먹이고 싶어지잖습니까!",
       "selected_2": "광대의 {{blacephalonName}}[[가]] 무언가 사용한 것 같다!\n{{switchPokemon}}의 모든 아이템이 랜덤하게 재조정된다!",
       "selected_3": "당황한 바보같은 표정! 완벽한 저의 한 수를 즐겨주시길!"
     },
     "3": {
       "label": "조롱을 되돌려준다",
-      "tooltip": "(-) 광대가 화를 냄\n(?) 포켓몬의 타입 조정",
+      "tooltip": "(-) 멤버 포켓몬의 타입 조정",
```

```suggestion
      "tooltip": "(-) 멤버 포켓몬들의 타입 조정",
```
원문도 Pokémons'다 보니 여러 마리에 영향 주는 걸 확실하게 나타내 주는 게 좋을 듯합니다